pendingTranslations.ftl

```diff
@@ -22,3 +22,77 @@ toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
+
+# Icon alts / aria-labels
+chevron-down-alt = Drop down button
+open-in-new-tab-aria-label = Open link in a new tab
+open-in-new-tab-alt = Open button
+
+# Status Pill
+
+status-pill-action-needed = Action Needed
+status-pill-progress = In Progress
+status-pill-fixed = Fixed
+
+# Exposure Card
+
+exposure-card-family-members = Family members
+exposure-card-email = Email
+exposure-card-phone-number = Phone number
+exposure-card-address = Address
+# Variables:
+# $exposure_num is the number of exposures found for a particular type, e.g. 3 found 
+exposure-card-num-found = 
+  { $exposure_num ->
+     *[other] { $exposure_num } found
+  }
+exposure-card-description-info-for-sale-part-one = This site is selling and publishing <data_broker_link>details about you</data_broker_link>
+exposure-card-description-info-for-sale-part-two = Remove this profile to protect your privacy.
+# Variables:
+# $data_breach_company is the company assosciated with the data breach.
+# $data_breach_date is the date of the data breach.
+exposure-card-description-data-breach-part-one = Your information was exposed in the <data_breach_link>{ $data_breach_company } data breach on { $data_breach_date }</data_breach_link>
+exposure-card-description-data-breach-part-two = We'll walk you through the steps to fix it.
+exposure-card-your-exposed-info = Your exposed info
+exposure-card-exposure-type-data-broker = Info for sale
+exposure-card-exposure-type-data-breach = Data breach
+
+# Dashboard
+
+dashboard-top-banner-protect-your-data-title = Let's protect your data
+# Variables:
+# $data_breach_total_num is the total number of data breaches the user has.
+# $data_broker_total_num is the total number of data brokers selling the user's data.
+dashboard-top-banner-protect-your-data-description = We found your data in { $data_breach_total_num } data breaches and { $data_broker_total_num } sites selling your personal info. We'll guide you on how to fix it.
+dashboard-top-banner-protect-your-data-cta = Let's fix it
+
+dashboard-top-banner-monitor-protects-your-even-more-title = Monitor now protects you even more
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan.
+dashboard-top-banner-monitor-protects-your-even-more-description = We can now find exposures of your personal info on { $data_broker_sites_total_num } data broker sites that publish and sell your personal info for a profit.
```

Comment:
Same as the previous comment. If there is a plural, please use the structure with `*[other]` to trigger the plural UI (there are multiple strings in this patch).